### PR TITLE
sim & robot.sh improvement

### PR DESCRIPTION
**Pull request by noah710**
set environment variables for local deployment 
launch continuous_path in local.launch
stop wiggling when the robot gets to the next point

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Make sure you
   - If you get a request timeout after the NUC has been powered on for a minute, power cycle the NUC
 
 ### I want to deploy the stack locally (magellan_sim)
```
Comment (nls5260) [inline README.md:43]: I would add more information here about the specifics about deploying locally.

File: robot.sh [hunk 2]
```diff
@@ -87,10 +113,17 @@ case $1 in
...
-        ssh ras@${ROBOT_IP}
+        if [[ -n "${LOCAL}" ]]; then
+            echo "You're sshing into your own machine!"
+        else
+            ssh ras@${ROBOT_IP}
+        fi
 	;;
 
+    "")
+        printf "Please select an action\n"
```
Comment (asaba96) [inline robot.sh]: Can we change this message to print the help/usage information?

File: robot.sh [hunk 3]
```diff
@@ -87,10 +113,17 @@ case $1 in
...
+      ;;
     *)
-        echo "Usage: robot.sh [start|stop|deploy|watch|shell|deploy-teensy|ssh]"
+        printf "Usage: ./robot.sh -a [start|stop|deploy|watch|shell|deploy-teensy|ssh] [-l]\n-l: deploy locally\n"
     ;;
-esac
+esac
```
Comment (asaba96) [inline robot.sh]: No EOF new line

**Review by asaba96 — APPROVED** [review]
LGTM

**Pull request merged**